src/cogent3/parse/genbank.py

```diff
@@ -348,85 +378,132 @@ def __str__(self):
         try:
-            return int(self._data)
+            return int(self._data) - 1
         except TypeError:
-            return self._data[0].first()
+            return self._data[0].start()
 
-    def last(self):
+    def stop(self):
         """Returns last base self could be."""
         try:
-            return int(self._data)
+            return int(self._data) - 1
         except TypeError:
-            return self._data[-1].last()
+            return self._data[-1].stop()
 
+    def first(self):
+        from cogent3.util.warning import deprecated
+
+        deprecated(
+            "method",
+            "Location.first",
+            "Location.start",
+            "2023.06.15",
+            "Warning: '.start()' is changed to reflect 0-based coordinated, previous implementation reflected 1-based coordinates",
+        )
 
-class LocationList(list):
-    """List of Location objects.
+        return self.start()
 
-    WARNING: Coordinates are based on 1, not 0, to match GenBank format.
-    """
+    def last(self):
+        from cogent3.util.warning import deprecated
+
+        deprecated(
+            "method",
+            "Location.last",
+            "Location.stop",
+            "2023.06.15",
+            "Warning: '.stop()' is changed to reflect 0-based coordinated, previous implementation reflected 1-based coordinates",
+        )
+        return self.stop()
+
+
+class LocationList(list):
+    """List of Location objects."""
 
     BIGNUM = 1e300
 
-    def first(self):
+    def start(self):
         """Returns first base of self."""
         curr = self.BIGNUM
         for i in self:
-            first = i.first()
-            if curr > first:
-                curr = first
+            start = i.start()
+            if curr > start:
+                curr = start
         return curr
 
-    def last(self):
+    def stop(self):
         """Returns last base of self."""
         curr = 0
         for i in self:
-            last = i.last()
-            if last > curr:
-                curr = last
+            stop = i.stop()
+            if stop > curr:
+                curr = stop
         return curr
 
+    def first(self):
+        from cogent3.util.warning import deprecated
+
+        deprecated(
+            "method",
+            "LocationList.first",
+            "LocationList.start",
+            "2023.06.15",
+            "Warning: '.start()' is changed to reflect 0-based coordinated, previous implementation reflected 1-based coordinates",
+        )
+
+        return self.start() + 1
+
+    def last(self):
+        from cogent3.util.warning import deprecated
+
+        deprecated(
+            "method",
+            "LocationList.last",
+            "LocationList.stop",
+            "2023.06.15",
+            "Warning: '.stop()' is changed to reflect 0-based coordinated, previous implementation reflected 1-based coordinates",
+        )
+        return self.stop() + 1
+
     def strand(self):
```

Comment:
can you change this to being a property too. It will still work as a method, but at least it can be just used as `<obj>.strand`